android/src/main/java/io/parity/signer/components/networkicon/IdentIconImage.kt

```diff
@@ -0,0 +1,91 @@
+package io.parity.signer.components.networkicon
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Column
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import io.parity.signer.components.networkicon.blockies.BlockiesIcon
+import io.parity.signer.components.networkicon.dot.DotIcon
+import io.parity.signer.ui.helpers.PreviewData
+import io.parity.signer.ui.theme.SignerNewTheme
+import io.parity.signer.uniffi.Identicon
+
+/**
+ * Just draw a standard IdentIcon used everywhere, with standard size
+ */
+@Composable
+fun IdentIconImage(
+	identIcon: Identicon,
+	modifier: Modifier = Modifier,
+	size: Dp = 28.dp
+) {
+	when (identIcon) {
+		is Identicon.Blockies -> {
+			BlockiesIcon(
+				seed = identIcon.identity,
+				preferedSize = size,
+				modifier = modifier,
+			)
+		}
+
+		is Identicon.Dots -> {
+			DotIcon(seed = identIcon.identity,
+				size = size,
+				modifier = modifier,
+				)
+		}
+//		is ImageContent.Svg -> {//will be used for another type of icons
```

Comment:
Q: Shouldn't we just delete it, if we need to bring it back, we can use git history?